src/Microsoft.AspNet.Razor.Runtime/DefaultTagHelperContent.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Default implementation of <see cref="TagHelperContent"/>.
+    /// </summary>
+    public class DefaultTagHelperContent : TagHelperContent, ITextWriterCopyable
+    {
+        private readonly BufferEntryCollection _buffer;
+
+        /// <summary>
+        /// Instantiates a new instance of <see cref="DefaultTagHelperContent"/>.
+        /// </summary>
+        public DefaultTagHelperContent()
+        {
+            _buffer = new BufferEntryCollection();
+        }
+
+        /// <inheritdoc />
+        public override bool IsModified
```

Comment:
- there's exactly one use of property expressions across Razor and MVC repos
- I asked why because no C# 6.0 feature is `var` and your comment was apparently not a teensy nit.  some C# 6.0 features do useful things (like consolidate information that would otherwise exist in two parts of the class) and there is a push to use those features in new code.  but there's no push for this feature.